packages/deploy/utils/lazyMinting.ts

```diff
@@ -0,0 +1,357 @@
+
+  const backendAuthWallet = new ethers.Wallet(
+    '0x4242424242424242424242424242424242424242424242424242424242424242'
+  );
+
+  const sigData = {
+    types: {
+      LazyMint: [
+        {name: 'caller', type: 'address'},
+        {name: 'creator', type: 'address'},
+        {name: 'nonce', type: 'uint16'},
+        {name: 'tier', type: 'uint8'},
+        {name: 'amount', type: 'uint256'},
+        {name: 'unitPrice', type: 'uint256'},
+        {name: 'paymentToken', type: 'address'},
+        {name: 'metadataHash', type: 'string'},
+        {name: 'maxSupply', type: 'uint256'},
+      ],
+    },
+    domain: {
+      name: 'Sandbox Asset Create',
+      version: '1.0',
+      chainId: network.config.chainId,
+      verifyingContract: await AssetCreateContract.getAddress(),
+    },
+    message: {
+      caller,
+      creator,
+      nonce,
+      tier,
+      amount,
+      unitPrice,
+      paymentToken,
+      metadataHash,
+      maxSupply,
+    },
+  };
+
+  const signature = await backendAuthWallet.signTypedData(
+    sigData.domain,
+    sigData.types,
+    sigData.message
+  );
+  return signature;
+};
+
+export const createMultipleLazyMintSignature = async (
+  data: LazyMintBatchData,
+  AssetCreateContract: Contract,
+  network: Network
+) => {
+  const {
+    caller,
+    creators,
+    tiers,
+    amounts,
+    unitPrices,
+    paymentTokens,
+    metadataHashes,
+    maxSupplies,
+  } = data;
+  const nonce = await AssetCreateContract.signatureNonces(caller);
+  const backendAuthWallet = new ethers.Wallet(
+    '0x4242424242424242424242424242424242424242424242424242424242424242'
+  );
+
+  const sigData = {
+    types: {
+      LazyMintBatch: [
+        {name: 'caller', type: 'address'},
+        {name: 'creators', type: 'address[]'},
+        {name: 'nonce', type: 'uint16'},
+        {name: 'tiers', type: 'uint8[]'},
+        {name: 'amounts', type: 'uint256[]'},
+        {name: 'unitPrices', type: 'uint256[]'},
+        {name: 'paymentTokens', type: 'address[]'},
+        {name: 'metadataHashes', type: 'string[]'},
+        {name: 'maxSupplies', type: 'uint256[]'},
+      ],
+    },
+    domain: {
+      name: 'Sandbox Asset Create',
+      version: '1.0',
+      chainId: network.config.chainId,
+      verifyingContract: await AssetCreateContract.getAddress(),
+    },
+    message: {
+      caller,
+      creators,
+      nonce,
+      tiers,
+      amounts,
+      unitPrices,
+      paymentTokens,
+      metadataHashes,
+      maxSupplies,
+    },
+  };
+
+  const signature = await backendAuthWallet.signTypedData(
+    sigData.domain,
+    sigData.types,
+    sigData.message
+  );
+  return signature;
+};
+
+export const giveSandToAccount = async (
+  SandContract: Contract,
+  account: string,
+  amount: bigint
+) => {
+  // Give sand to lazyMintingTestAccount1
+  // impersonate CHILD_CHAIN_MANAGER
+  await ethers.provider.send('hardhat_impersonateAccount', [
+    '0x8464135c8F25Da09e49BC8782676a84730C318bC',
```

Comment:
Would prefer not to hardcode accounts if possible?

Reply:
This is a very good point, updating.